File: COMPILADOR/kokoslan_/src/src/java/exception/KoKoNotFoundId.java
/**
 @author loriacarlos@gmail.com
 
*/
package src.java.exception;

import src.java.ast.KoKoId;

public class KoKoNotFoundId extends RuntimeException {
	public KoKoNotFoundId(KoKoId id){
		super( String.format("*** Id '%s' is not defined! ***", id.getValue()) );
	}
	
}

File: COMPILADOR/kokoslan/src/KoKoKotlin/exception/KoKoNotFoundId.kt
/**
 * @author loriacarlos@gmail.com
 */
package KoKoKotlin.exception

import KoKoKotlin.ast.KoKoId


class KoKoNotFoundId(id: KoKoId) :
        RuntimeException(String.format("*** Id '%s' is not defined! ***", id.value))